src/commands/app/__tests__/create.test.ts

```diff
@@ -0,0 +1,112 @@
+import AppCreate from 'commands/app/create';
+import { APP_VERSION_STATUS } from 'consts/app-versions';
+import {
+  createAppUrl,
+  getCreateAppFeatureReleaseUrl,
+  getCreateAppFeatureUrl,
+  getTunnelingDomainUrl,
+  listAppVersionsByAppIdUrl,
+} from 'consts/urls';
+import { buildMockFlags, getRequestSpy, getStdout, mockListPromptImplementation } from 'test/cli-test-utils';
+
+const requestSpy = getRequestSpy();
+
+describe('app:create', () => {
+  jest.setTimeout(1_000_000);
+  const MOCK_BASE_RESPONSE = { headers: {} };
+  const MOCK_APP_ID = 1_000_000_001;
+  const MOCK_APP_VERSION_ID = 1_000_000_002;
+  const MOCK_APP_FEATURE_ID = 1_000_000_003;
+  const question = 'Select a template to start with';
+  const answer = 'Fullstack React + Node.js (TypeScript)';
```

Comment:
This should be a part of a specific `it` no? Every test case in here will go with the same set of `question` & `answer`